systems-at-scale.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: How to Instrument Systems at Scale
+---
+_Before getting onto recommendations on how to instrument your large-scale system with OpenTracing, be sure to read the [Specification overview](/spec)._
+
+<div id="toc"></div>
+
+
+## Spans and Relationships
+
+The two fundamental aspects of implementing OpenTracing across your infrastructure are _Spans_ and the _Relationships_ between those spans:
+
+* **_Spans_** are a logical unit of work in the system that has a start time and a duration. In a trace Spans are the components across the systems exercised on a specified path.
```

Comment:
@pritianka  :+1: I think getting the terminology consistent will be a huge help as the OT community grows.